includes/fields/class-gravityview-field-list.php

```
@@ -234,6 +236,40 @@ public static function get_column_label( GF_Field_List $field, $column_id, $back
+	 * @since TBD
+	 *
+	 * @param string   $return The current merge tag value to be filtered.
+	 * @param string   $raw_value The raw value submitted for this field. May be CSV or JSON-encoded.
+	 * @param string   $value The original merge tag value, passed from Gravity Forms
+	 * @param string   $merge_tag If the merge tag being executed is an individual field merge tag (i.e. {Name:3}), this variable will contain the field's ID. If not, this variable will contain the name of the merge tag (i.e. all_fields).
+	 * @param string   $modifier The string containing any modifiers for this merge tag. For example, "maxwords:10" would be the modifiers for the following merge tag: `{Text:2:maxwords:10}`.
+	 * @param GF_Field $field The current field.
+	 *
+	 * @return string
+	 */
+	public function handle_list_column_modifier( $return, $raw_value, $value, $merge_tag, $modifier, $field ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$value'. (Unused Code Rules)

(UnusedFormalParameter)